containers/unit_tests/TestDualView.hpp

```diff
@@ -622,6 +622,16 @@ TEST(TEST_CATEGORY,
             const_dv.view<Kokkos::DefaultExecutionSpace>());
 }
 
+TEST(TEST_CATEGORY, dualview_sequential_host_init) {
+  Kokkos::DualView<double*> dv(
+      Kokkos::view_alloc("myView", Kokkos::SequentialHostInit), 100u);
```

Comment:
In our cuda build, this line doesn't compile. 

Because the constructor of `DualView` forwards the properties to the constructor of the device view:

- https://github.com/kokkos/kokkos/blob/3fe17d96d42eb2bd84f1f2a835a2ed0ba87606b6/containers/src/Kokkos_DualView.hpp#L279

And `SequentialHostInit` cannot be used for a device view:

- https://github.com/kokkos/kokkos/blob/3fe17d96d42eb2bd84f1f2a835a2ed0ba87606b6/core/src/View/Kokkos_ViewAlloc.hpp#L202-L206